test/e2e/network/netpol/network_policy.go

```diff
@@ -1322,6 +1323,10 @@ func initializeResources(f *framework.Framework) error {
 	}
 
 	framework.Logf("finished initializing cluster state")
-
-	return k8s.waitForHTTPServers(model)
+	if framework.NodeOSDistroIs("windows") {
+		protocols = append(protocols, v1.ProtocolTCP)
+	} else {
+		protocols = model.Protocols
```

Comment:
I didn't went through the code, what does this really mean?

Reply:
we only test TCP for windows because UDP polling on windows will crash the nodes.
for linux, we test all protocols